pcdsdevices/mpodApalis.py

```diff
@@ -0,0 +1,248 @@
+import logging
+from ophyd.device import Component as Cpt
+from ophyd.device import Device
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+from pcdsdevices.interface import BaseInterface
+import time
+logger = logging.getLogger(__name__)
+
+
+class MPODApalisChannel(BaseInterface, Device):
+    """
+    MPODApalis Channel Object.
+
+    Parameters
+    ----------
+    channel_prefix : str
+        The EPICS base of the MPOD Channel. E.g.: `TMO:MPOD:01:M5:C6
+    name : str
+       A name to refer to the device.
```

Comment:
Misaligned docstring
```suggestion
    channel_prefix : str
        The EPICS base of the MPOD Channel. E.g.: `TMO:MPOD:01:M5:C6
    name : str
        A name to refer to the device.
```